consensus/fork_choice/src/fork_choice.rs

```diff
@@ -318,6 +325,11 @@ pub struct ForkChoice<T, E> {
     queued_attestations: Vec<QueuedAttestation>,
     /// Stores a cache of the values required to be sent to the execution layer.
     forkchoice_update_parameters: ForkchoiceUpdateParameters,
+
+    /// Fast Confirmation Rule
+    #[cfg(feature = "fast_confirmation")]
+    fast_confirmation: FastConfirmation<E>,
```

Comment:
This field could be `Option<FastConfirmation<E>>` with `None` representing the case where it is turned off at runtime.